selfdrive/car/hyundai/hyundaican.py

```diff
@@ -95,10 +95,10 @@ def create_acc_commands(packer, enabled, accel, jerk, idx, lead_visible, set_spe
   commands.append(packer.make_can_msg("SCC11", 0, scc11_values))
 
   scc12_values = {
-    "ACCMode": 1 if enabled else 0,
+    "ACCMode": 2 if enabled and gas_pressed else 1 if enabled else 0,
```

Comment:
If this does correspond to gas pressed, should we include the buffer after gas is released?